utils/node_local_test.py

```diff
@@ -145,6 +146,14 @@ def __init__(self, filename):
         self._file = __file__.lstrip('./')
         self._flush()
 
+        if junit:
+            test_case = junit_xml.TestCase('Startup', classname='NLT')
+            self.ts = junit_xml.TestSuite('Node Local Testing', test_cases=[test_case],
+            self.tc = junit_xml.TestCase('Sanity', classname='NLT.Sanity')
```

Comment:
(pylint-syntax-error) expression cannot contain assignment, perhaps you meant "=="? (<unknown>, line 159)